cubics-obc-test/Components/DeployablesInterface/DeployablesInterface.cpp

```diff
@@ -0,0 +1,83 @@
+// ======================================================================
+// \title  DeployablesInterface.cpp
+// \author fprime-dev
+// \brief  cpp file for DeployablesInterface component implementation class
+// ======================================================================
+
+
+#include <Components/DeployablesInterface/DeployablesInterface.hpp>
+#include <FpConfig.hpp>
+
+namespace Components {
+
+  // ----------------------------------------------------------------------
+  // Construction, initialization, and destruction
+  // ----------------------------------------------------------------------
+
+  DeployablesInterface ::
+    DeployablesInterface(
+        const char *const compName
+    ) : DeployablesInterfaceComponentBase(compName)
+  {
+    openSocket();
+  }
+
+  DeployablesInterface ::
+    ~DeployablesInterface()
+  {
+    closeSocket();
+  }
+
+  int DeployablesInterface :: openSocket(){
```

Comment:
If all the Components are connecting with TCP you might want to make this a common utility

Reply:
I am working on this now, although I am not sure how to modify the Fprime build / compile process to compile a seperate 'TCP_stuff' source file and link it , given it will not be an 'fprime' component.